addons/l10n_it_stock_ddt/tests/test_ddt.py

```diff
@@ -27,10 +27,15 @@ def setUpClass(cls, chart_template_ref='l10n_it.l10n_it_chart_template_generic')
             'street': 'Piazza Guglielmo Marconi 5',
             'zip': '26100',
             'city': 'Cremona',
-            'country_id': cls.env.ref('base.it'),
+            'country_id': cls.env.ref('base.it').id,
             'vat': 'IT12345670124'
         })
 
+        settings = cls.env['res.config.settings'].create({})
+        if hasattr(settings, 'button_create_proxy_user'):
+            # Needed when `l10n_it_edi_sdiscoop` is installed
+            settings.button_create_proxy_user()
```

Comment:
It is weird that you put this here as sdicoop does not do anything with what is done in this module.  (except a field in the xml)

Reply:
The test has to be run post install, and sdiscoop might or might not be installed

Reply:
<img width="1173" alt="Captura de Pantalla 2021-07-09 a la(s) 10 23 58" src="https://user-images.githubusercontent.com/382893/125084426-e87f2280-e09f-11eb-9440-ec7719ca8333.png">

Reply:
needed for action_post